datafusion/physical-expr/src/aggregate/array_agg_distinct.rs

```diff
@@ -308,146 +304,4 @@ mod tests {
 
         check_merge_distinct_array_agg(col1, col2, expected, DataType::Int32)
     }
-
-    #[test]
-    fn distinct_array_agg_nested() -> Result<()> {
```

Comment:
I don't understand the rationale for deleting these tests -- is the functionality covered elsewhere? If we deleted this code, won't we have less test coverage?

Reply:
@alamb 
Since update_batch() of DistinctArrayAggAccumulator only accepts non-list after the change. Nested array in these tests is not allowed.
Should I just delete the test or add an assertion to ensure we get Err in this case?